app/src/main/java/fr/free/nrw/commons/mwapi/ApacheHttpClientMediaWikiApi.java

```diff
@@ -436,6 +447,88 @@ public String wikidataCreateClaim(String entityId, String property, String snakt
         return null;
     }
 
+    /**
+     * Edits claim using the commons API by adding P180 tag for an image
+     * https://commons.wikimedia.org/wiki/Wikibase/API
+     * @param entityId the commons api entity to be edited
+     * @param fileEntityId entity id for file, page id
+     * @return returns revisionId if the claim is successfully created else returns null
+     * @throws IOException
+     */
+    @Nullable
+    @Override
+    public String wikidataEditEntity(String entityId, String fileEntityId) throws IOException {
```

Comment:
Why does the name of this method include `wikidata`? The P180 property must be sent to Commons' Wikibase server, not to the Wikidata server, right? Would not a name like `commonsWikibaseEditEntity` or `wikibaseEditEntity` make more sense? If it is for consistency with the existing methods, feel free to rename them from `wikidata` to `wikibase` _(up to now we did not connect to any Wikibase server other than the Wikidata server, which explains the current wording)_. Thanks :-)

Reply:
Sure.